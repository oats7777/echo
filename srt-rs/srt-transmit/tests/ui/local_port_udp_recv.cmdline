["udp://:4001?local_port=2223" ,  "udp://127.0.0.1:4000"]