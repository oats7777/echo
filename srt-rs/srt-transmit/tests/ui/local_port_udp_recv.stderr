Invalid settings detected: local_port is incompatiable with udp listen mode

See srt-transmit --help for more info
